### ISSUE TITLE
change classes

### ISSUE THREAD

**Issue by kyriefeng** [felixchenfy/Realtime-Action-Recognition#51]
how can i change the classes?
Expect make some changes in config,should i put some new trainning data about new classes?

**Comment by felixchenfy** [comment]
In the config file, the class is defined as `classes: ['stand', 'walk', 'run', 'jump', 'sit', 'squat', 'kick', 'punch', 'wave']`. You can change it. And then you need to add your own data.

**Comment by kyriefeng** [comment]
thanks for reply !there is some dataset you recommended ?or should i make videos and divide them into some frames ?

**Comment by felixchenfy** [comment]
If you are doing a course project, I would encourage you to collect your own video a make a small dataset.

If you are doing a research project, definitely look for other datasets that are used by famous papers. And also look for the github projects that are supported by those papers. Don't use mine, because mine is really not good.
I don't have any recommendations for the dataset. I didn't use another dataset before.

**Comment by kyriefeng** [comment]
when i add my own training data to this project ,and rename the folder ,such as jump1.but there is an error :ValueError: invalid literal for int() with base 10: 'jump1'
is something wrong about folder name ?and what's the meaning of this folder name :jump_03-02-12-34-01-795?the meaning of these numbers ?

**Comment by felixchenfy** [comment]
Q: "jump_03-02-12-34-01-795?the meaning of these numbers ?"
A: The number is the date that I recorded this image. However, the number does NOT matter. Only the first word (which is "jump" here) is used as the class type.

Q: "ValueError: invalid literal for int() with base 10: 'jump1'"
A: Sorry, I really don't know the root cause by looking at this error.

About the data format, I have documented it in the README:
![image](https://user-images.githubusercontent.com/22253529/80661358-4f885c80-8a43-11ea-8747-74c14ccc712d.png)
